deployment/run.sh

```diff
@@ -24,6 +24,19 @@ es_ops_node_quorum=${ES_OPS_NODE_QUORUM:-$((es_ops_cluster_size/2+1))}
 es_ops_recover_after_nodes=${ES_OPS_RECOVER_AFTER_NODES:-$((es_ops_cluster_size-1))}
 es_ops_recover_expected_nodes=${ES_OPS_RECOVER_EXPECTED_NODES:-$es_ops_cluster_size}
 es_ops_recover_after_time=${ES_OPS_RECOVER_AFTER_TIME:-5m}
+# node selectors
+es_ns_key=$(echo $ES_NODESELECTOR | cut -f1 -d=)
+es_ns_value=$(echo $ES_NODESELECTOR | cut -f2 -d=)
```

Comment:
#63 contains the function above, after that merges I'll reuse it for this branch